exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -7,6 +7,7 @@
 
 import io.opentelemetry.api.metrics.MeterProvider;
 import io.opentelemetry.exporter.internal.ExporterMetrics;
+import io.opentelemetry.exporter.internal.InstrumentationUtil;
```

Comment:
```suggestion
import io.opentelemetry.api.internal.InstrumentationUtil;
```

Should be updated to use the new API introduced in https://github.com/open-telemetry/opentelemetry-java/pull/6546